sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -201,6 +205,54 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       settingKey[Boolean]("Whether webpack should emit source maps at all")
 
     /**
+      * Additional directories, monitored for webpack launch.
+      *
+      * Changes to files in these directories that match
+      * `webpackMonitoredIncludeFilter` enable webpack launch in
+      * `webpack` task.
+      *
+      * Defaults to an empty `Seq`.
+      * 
+      * @group settings
+      * @see [[webpackMonitoredIncludeFilter]]
+      */
+    val webpackMonitoredDirectories: SettingKey[Seq[File]] =
+      settingKey[Seq[File]]("Directories, monitored for webpack launch")
+
+    /**
+      * Filter for files, monitored for webpack launch.
+      * 
+      * Changes to files matching this filter in `webpackMonitoredDirectories`
+      * enable webpack launch in `webpack` task.
+      * 
+      * Defaults to `AllPassFilter`
+      * 
+      * @group settings
+      * @see [[webpackMonitoredDirectories]]
+      */
+    val webpackMonitoredIncludeFilter: SettingKey[FileFilter] =
+      settingKey[FileFilter]("Filter for files, monitored for webpack launch")
```

Comment:
You're right, of course. Scoped `includeFilter` is more idiomatic. Will change this shortly.